vanilla-framework/vanilla-framework-react
# Update Vanilla React Storybook

# Done
- Updated to Vanilla 1.6.3 and fixed associated bugs with List and SteppedList scss
- Added storybook knobs addon and added knobs for most components
- Added storybook options addon and changed default name + link (in top left), and knob panel auto-shwoing on right
- Refactored Link and CodeBlock components to accept booleans instead of modifier prop

# QA
- Pull code
- Run `yarn lint` and `yarn test` to ensure there are no linting or testing errors
- Run `./run serve --watch` and navigate to http://localhost:8102/
- Check that all the components still work fine (especially the Links, which were changed a bit in 1.6.3)
- Play around with the knobs panel and see if everything works
